spec/system/publishers/publishers_can_publish_a_vacancy_as_a_school_spec.rb

```diff
@@ -2,34 +2,28 @@
 
 RSpec.describe "Creating a vacancy" do
   let(:publisher) { create(:publisher) }
-  let(:school) { create(:school, :not_applicable, name: "Salisbury School", safeguarding_information: nil) }
+  let(:vacancy) do
+    VacancyPresenter.new(build(:vacancy,
+                               :ect_suitable,
+                               job_roles: ["teacher"],
+                               phases: %w[secondary],
+                               key_stages: %w[ks3],
+                               publish_on: Date.current))
+  end
+  let(:created_vacancy) { Vacancy.last }
 
   before { login_publisher(publisher: publisher, organisation: school) }
+  after { logout }
 
-  scenario "Visiting the school page" do
-    visit organisation_jobs_with_type_path
-
-    expect(page).to have_content("Salisbury School")
-
-    click_on I18n.t("buttons.create_job")
-
-    expect(page).to have_content(I18n.t("jobs.create_job_caption", step: 1, total: 4))
-  end
-
-  context "creating a new vacancy" do
-    let(:vacancy) do
-      VacancyPresenter.new(build(:vacancy,
-                                 :ect_suitable,
-                                 job_roles: ["teacher"],
-                                 phases: %w[secondary],
-                                 key_stages: %w[ks3],
-                                 publish_on: Date.current))
-    end
-    let(:created_vacancy) { Vacancy.last }
+  context "non-faith school" do
+    let(:school) { create(:school, :not_applicable, name: "Salisbury School") }
 
-    scenario "follows the flow" do
+    it "follows the flow" do
```

Comment:
:question: out of curiosity. What criteria are you applying to set `it` vs `scenario` for the feature tests?

We have a tech debt ticket (and a pending team discussion on it) regarding normalising the feature specs `it/scenario`, but I see you're using both on the same feature specs file. So I wonder if you apply a particular rule regarding whether to use `it` or `scenario` in each case.